keyboards/skyloong/gk75/q1/mcuconf.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2023 JZ-Skyloong (@JZ-Skyloong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef STM32_I2C_USE_I2C1
+#define STM32_I2C_USE_I2C1 TRUE
+
+#undef STM32_I2C_USE_DMA
+#define STM32_I2C_USE_DMA FALSE
```

Comment:
```suggestion
```
The I2Cv1 driver in ChibiOS, which is used for STM32F103, does not have the `STM32_I2C_USE_DMA` option — only I2Cv2 and I2Cv3 drivers support it.